src/automaticassessmentconfiguration.cs

```diff
@@ -21,13 +22,67 @@ namespace AutomaticAssessmentConfiguration {
 	public enum AEAssessmentErrorCode : long {
 		Unknown = 1
 	}
+
+	[Unavailable (PlatformName.MacCatalyst)]
+	[iOS (14, 0)]
+	[Advice ("This API is not available when using UIKit on macOS.")]
+	[Native]
+	enum AEAutocorrectMode : long {
+		None = 0,
+		Spelling = 1 << 0,
+		Punctuation = 1 << 1,
+	}
 	
 	[Unavailable (PlatformName.MacCatalyst)]
 	[Mac (10,15,4), iOS (13,4)]
 	[Advice ("This API is not available when using UIKit on macOS.")]
 	[BaseType (typeof (NSObject))]
 	interface AEAssessmentConfiguration : NSCopying {
 
+		[NoMac, iOS (14, 0)]
+		[Advice ("This API is not available when using UIKit on macOS.")]
```

Comment:
This is not needed as the parent class already holds this advice

```suggestion
		[Advice ("This API is not available when using UIKit on macOS.")]
```

Reply:
same for all the ones below within `AEAssessmentConfiguration`